### PR TITLE
Feature/condo/doma 1469/update control room

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
- new table columns: `unit` and `classifier`
- new `address` and `date` render formats
- table max size now is 1600px
- table page size now is 30 objects
- tickets with the "open" status are pinned at the top of the table by default

infinity scroll will be made a separate task

изображение

### CODE REVIEW

**Review by AntonAL — CHANGES_REQUESTED** [review]
I've launched the app locally with your branch and tickets does't get's displayed in `/tickets` page.

**Comment by nomerdvadcatpyat** [comment]
> I've launched the app locally with your branch and tickets does't get's displayed in `/tickets` page.

did you run `migrate`?

File: apps/condo/domains/ticket/schema/Ticket.js [hunk 1]
```diff
@@ -59,7 +62,10 @@ const Ticket = new GQLListSchema('Ticket', {
             knexOptions: { isNotNullable: true }, // Required relationship only!
             kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
         },
-
+        order: {
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/Ticket.js:65]: Does this field overrides sorting order only at display phase in UI?

Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/Ticket.js:65]: No, this field is needed for server-side sorting

File: apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+--
+-- Add field orderTo to ticketchange
+--
+ALTER TABLE "TicketChange" ADD COLUMN "orderTo" integer NULL;
+--
+-- Add field order to tickethistoryrecord
+--
+ALTER TABLE "TicketHistoryRecord" ADD COLUMN "order" integer NULL;
+
+    UPDATE "Ticket" SET "order" = 100 WHERE "id" = '6ef3abc4-022f-481b-90fb-8430345ebfc2';
```
Comment (AntonAL) [inline apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js:24]: It looks strange, what the ticket with this `id` is about?

Comment (nomerdvadcatpyat) [reply apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js:24]: oops
It's a `open` status id.
Should be ` UPDATE "Ticket" SET "order" = 100 WHERE "status" = '6ef3abc4-022f-481b-90fb-8430345ebfc2';`

Comment (nomerdvadcatpyat) [reply apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js:24]: I want to update the `order` here for all `tickets` that have the `open` status.

Comment (AntonAL) [reply apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js:24]: Yes, it's correct.

**Comment by AntonAL** [comment]
> did you run `migrate` script?

Did migrate and it got displayed.

File: apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js (hunk 2 above)
Comment (nomerdvadcatpyat) [reply apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js]: regenerate migration with correct update in b91bc7f

Comment (AntonAL) [reply apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js]: Now it's correct.
Fix indentation of `UPDATE` statement, please, to be consistent.

**Review by sitozzz — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

**Pull request merged**